crates/sgx_validation/Cargo.toml

```diff
@@ -9,7 +9,10 @@ description = "Intel SGX Attestation validation library for Steward"
 cryptography = { path = "../cryptography" }
 anyhow = { version = "^1.0.55", default-features = false }
 der = { version = "0.6", features = ["std"], default-features = false }
+hex = "0.4"
```

Comment:
Why different import style than the rest of this file?

Reply:
Fixed